From e648bb2439cad0d2ce47875edc7944d0b446fdb4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 9 Apr 2025 14:38:31 -0700
Subject: [PATCH] Bump setuptools dependency from 74 to 79

Upstream sticks to 0.74 since setuptools have dropped msvc9compiler
support beyond that which is needed for python 3.9 on windows. We
do not have this problem on Linux

Upstream-Status: Inappropriate [OE-specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index dd885e4..9f273b3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
 # setuptools 64+ support --build-option
 # setuptools 74+ drops distutils.msvc9compiler required for Python 3.9 under Windows
-requires = ["setuptools>=64,<74"]
+requires = ["setuptools>=64,<79"]
 backend-path = ["build_script"]
 build-backend = "pyzstd_pep517"
